libafl/src/bolts/shmem.rs

```diff
@@ -84,6 +84,10 @@ impl ShMemDescription {
 
 /// An id associated with a given shared memory mapping ([`ShMem`]), which can be used to
 /// establish shared-mappings between proccesses.
+/// Id is a file descriptor if you use `MmapShMem` or `AshmemShMem`.
```

Comment:
Why a file descriptor for MmapShMem? I thought it's a file on disk?

Reply:
Yeah, but what we are actually storing into ShMemId.Id is the file descriptor atm. (else unix_shmem_server does not work)
The underlying file's name is not put into `Id` itself